solr/bin/solr.in.cmd

```diff
@@ -127,7 +127,7 @@ REM environments where security is not a concern, 0.0.0.0 can be used to allow
 REM Solr to accept connections on all network interfaces.
 REM set SOLR_JETTY_HOST=127.0.0.1
 REM Sets the network interface the Embedded ZK binds to.
-REM set SOLR_ZK_EMBEDDED_HOST=127.0.0.1
+REM set SOLR_ZOOKEEPER_SERVER_EMBEDDED_HOST=127.0.0.1
```

Comment:
why put the word "embedded" in there?  "server" implies embedded; it refers to the ZK server that Solr is running itself, embedded.
I could see swapping SERVER for EMBEDDED though... it would be clearer.

Reply:
We do want to keep this environmental name the same as the system property, so SOLR_ZOOKEEPER_EMBEDDED_HOST so we get the magic translation and don't need to pass it in expliiclty to Solr...

Reply:
I understand that; they need to be consistent for EnvUtils to work.
I'm discussing what the name should be (which will look a certain way when expressed as an env var vs system property).